### ISSUE TITLE
[Feat]: ability to link directly to a specific chart

### ISSUE THREAD

**Issue by andrewm4894** [netdata/netdata-cloud#777]
### Problem

When sharing a link with someone i would like for the link to take them directly to an individual chart. Im not sure this is possible right now. 

For example [this link](https://app.netdata.cloud/spaces/netdata-demo/rooms/all-nodes/overview#metrics_correlation=false&after=-900&before=0&utc=Europe%2FLondon&offset=%2B0&timezoneName=Edinburgh%2C%20London&modal=&modalTab=&d8a4e0c5-7c79-4145-900e-83a9f06fcb6a-overview-chartName=menu_prometheus_aws_cluster-autoscaler_submenu_apiserver_storage) will take you here:

![image](https://user-images.githubusercontent.com/2178292/227502103-5dc974d2-1c99-4696-bd36-df5521319dab.png)

But what if i actually want to link to the 3rd chart in that submenu - i don't think this is possible at the moment.

### Description

When a user interacts with a specific chart - add that to the url in such a way that if i copy and share that url when someone opens it, it will try take me to that specific chart. 

### Importance

really want

### Value proposition

1. make link sharing and collaboration a lot easier

**Comment by jacekkolasa** [comment]
Maybe we could create additional button, to copy link to the chart? It would be easier to do probably than syncing chart during scrolling.

@christophidesp @hugovalente-pm

**Comment by hugovalente-pm** [comment]
yes, that makes sense we even had some designs in the past that were considering that but doesn't this impact on the work @novykh is doing on the new charts?

![image](https://user-images.githubusercontent.com/82235632/227597332-904b0308-fb5d-46e4-ad63-95c662a04976.png)

wdyt @christophidesp ?

**Comment by christophidesp** [comment]
It makes sense to add the share link to the header.
